lib/Common.ts

```diff
@@ -136,7 +136,14 @@ export class MethodDate {
             }
             this.dateObject.setFullYear(year); // Force 2 digit year to be set correctly
         } else {
+            let today = new Date();
             this.dateObject = new Date();
+            this.dateObject.setFullYear(today.getFullYear());
```

Comment:
this seems unnecessary, why do we need this?

Reply:
it looks like we just need the setHours/setMinutes/setSeconds

Reply:
Agreed, removed sets